packages/agent/src/orchestrate/interface/orchestrateInterfaceOperations.ts

```diff
@@ -156,24 +165,22 @@ function createApplication<Model extends ILlmSchema.Model>(props: {
             "GET method should not have request body. Change method, or re-design the operation.",
           value: op.requestBody,
         });
-      op.authorizationRoles?.forEach((role, j) => {
-        if (props.roles === null) {
-          op.authorizationRoles = null;
-        }
-        if (props.roles?.find((it) => it === role) === undefined)
-          errors.push({
-            path: `operations[${i}].authorizationRoles[${j}]`,
-            expected: `undefined | ${props.roles?.join(" | ")}`,
-            description: [
-              `Role "${role}" is not defined in the roles list.`,
-              "",
-              "Please select one of them below, or do not define (undefined):  ",
-              "",
-              ...(props.roles ?? []).map((role) => `- ${role}`),
-            ].join("\n"),
-            value: role,
-          });
-      });
+      if (props.roles === null) {
+        op.authorizationRole = null;
+      }
```

Comment:
change this part to
```typescript

      if (props.roles === null) {
        op.authorizationRole = null;
      }

      if (op.authorizationRole === null) {
        return;
      }
```

because the props.roles (the roles that defined at the analyze agent) is null, `props.roles?.find((role) => role === op.authorizationRole) === undefined` is always true. this is an error. If `props.roles` is null, there are no role. so just return success.